labelbox/schema/dataset.py

```diff
@@ -54,13 +56,21 @@ def create_data_row(self, **kwargs):
         return self.client._create(DataRow, kwargs)
 
     def create_data_rows(self, items):
-        """ Creates multiple DataRow objects based on the given items.
-        Each element in `items` can be either a `str` or a `dict`. If
-        it's a `str`, then it's interpreted as a file path. The file
-        is uploaded to Labelbox and a DataRow referencing it is created.
-        If an item is a `dict`, then it should map `DataRow` fields (or their
-        names) to values. At the minimum it must contain a `DataRow.row_data`
-        key and value.
+        """ Creates multiple DataRow objects based on the given items. 
+        
+        Each element in `items` can be either a `dict` or a `str`. If an item is 
+        passed as a `dict`, it should map `DataRow` fields (or their names) to 
+        values.
+        If passing a path to an external URL, `items` must be passed as a 
+        `dict` containing `DataRow.row_data` as the key and the external
+        URL as the value. If the value given for `DataRow.row_data` is a 
+        `str`, then it is interpreted as a file path. The file is then uploaded 
+        to Labelbox and a DataRow referencing it is created.
+        If passing a path to a local file, `items` must be passed as a `str`.
```

Comment:
Please reformat so each line is as long as possible, up to 80 characters. I know, it's tedious :/